volatility3/framework/symbols/linux/utilities/modules.py

```diff
@@ -684,3 +688,82 @@ def get_requirements(cls):
             )
 
         return reqs
+
+
+class ModuleDisplayPlugin(plugins.PluginInterface):
```

Comment:
This seems like the wrong place for it, and inheriting from the plugin interface means it may get listed as main plugin (our code to list plugins looks for anything that inherits from plugin).  Are we sure that's what we want?  I'd much prefer any plugins we have to live *as* a plugin in the right directory.